spec/views/pages/index.html.erb_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe "pages/index.html.erb", type: :view do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
Could we add a scenario here? something like this:

```ruby
it 'displays the title' do
  render

  rendered.should contain('Smartflix')
end
```

Reply:
@the-harry I have added a similar scenario like this

```
it 'displays the title' do
    render

    expect(rendered).to have_text('Smartflix')
end
```